java/lib.jshell.agent/nbproject/project.properties

```diff
@@ -21,5 +21,5 @@ javac.compilerargs=-Xlint -Xlint:-serial
 extra.module.files=modules/ext/nb-custom-jshell-probe.jar,modules/ext/nb-mod-jshell-probe.jar
 jnlp.indirect.jars=modules/ext/nb-custom-jshell-probe.jar,modules/ext/nb-mod-jshell-probe.jar
 
-agentsrc.asm.cp=${libs.asm.dir}/core/asm-all-5.0.1.jar
+agentsrc.asm.cp=${libs.asm.dir}/core/asm-7.0.jar
```

Comment:
Yes, the agent needs all dependencies in its JAR, or on bootstrap classpath. I am afraid that cutting down the bundled library size would be painful: the ASM is actually used to inject `<classinit>` into user code so that the agent gets a reference to a `ClassLoader` that loaded the particular class of interest. Since I need to inspect the class contents and actually generate bytecode, most of the packages are IMHO used; maybe we can save `tree.analysis`.

The classloader can be later reach the class by name. This is useful for more arcane environments with custom classloaders (e.g. NetBeans itself ;))